src/css/stars.css

```diff
@@ -0,0 +1,38 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 24px;
+}
+
+.stars__list {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  display: flex;
+  margin-right: 4px;
+
+  height: 16px;
+  width: 16px;
+
+  background-image: url(../images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin: 0;
+  background-image: url(../images/star.svg);
```

Comment:
Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.